package.json

```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@dcos/http-service": "0.2.1",
-    "@dcos/mesos-client": "0.1.10",
+    "@dcos/mesos-client": "0.2.0",
```

Comment:
We might want bumping that one to `0.2.1` to include stuff we have in `http-client@0.3.0` which is pretty important https://github.com/dcos-labs/http-service/compare/v0.2.1...v0.3.0

Reply:
@nLight could you please open a PR to update the dependency? https://jira.mesosphere.com/browse/DCOS_OSS-2400

Reply:
https://github.com/dcos/dcos-ui/pull/2922